gno.land/pkg/integration/testscript_gnoland.go

```diff
@@ -193,6 +193,7 @@ func SetupGnolandTestscript(t *testing.T, p *testscript.Params) error {
 
 		genesis := gnoland.DefaultGenState()
 		genesis.Balances = LoadDefaultGenesisBalanceFile(t, gnoRootDir)
+		genesis.Bank.Params.TotalSupply = gnoland.TotalBalance(genesis.Balances)
```

Comment:
from what i can read, this behavior is specific to test scripts, but basically it makes any token in the genesis becoming a non-growing one, while i believe this should not be the case.